### ISSUE TITLE
quotes in integer module documentation

### ISSUE THREAD

**Issue by jlaurens** [latex3/latex3#1477]
The documentation is using various quotes.

- apostrophe (U+0027) `'` for octal numbers, displayed as right single quotation mark `’` (U+2019)
- grave accent `` ` `` (U+0060) for character code, displayed as left single quotation mark `‘` (U+2018)
- quotation mark `"` displayed as... quotation mark

Something like `\textquotesingle` and `\textasciigrave` for the former is more appropriate.

**Comment by josephwright** [comment]
Isn't that just the normal font situation?

**Comment by davidcarlisle** [comment]
> Isn't that just the normal font situation?

I don't think we should change the markup but the font situation is more difficult now than it was, as while the font is the same, and the characters really are ascii quotes, they now get a unicode mapping to typeset left and right quotes

If I grab a bit from the bottom of the note on page 166 of interface3 I get

https://w3c.github.io/xml-entities/unicode-names.html?%E2%80%98%E2%9F%A8char%20%E2%9F%A9%20or%20%E2%80%98%5C%E2%9F%A8char%20%E2%9F%A9

showing U+2018 `‘` not  U+0060  `'`

We could (I suppose?) generally adjust the unicode mapping for the tt font used in documentation so the ascii quote characters map to themselves.
